Source/WebCore/loader/LinkLoader.h

```diff
@@ -54,6 +54,7 @@ struct LinkLoadParameters {
     String imageSizes;
     String nonce;
     ReferrerPolicy referrerPolicy { ReferrerPolicy::EmptyString };
+    RequestPriority fetchPriorityHint;
```

Comment:
Probably does not hurt to initialize it.

Reply:
Sure, done.